app/scripts/components/common/map/style-generators/raster-timeseries.tsx

```diff
@@ -272,14 +272,36 @@ export function RasterTimeseries(props: RasterTimeseriesProps) {
         LOG && console.log('Payload', payload);
         LOG && console.groupEnd();
         /* eslint-enable no-console */
+        
+        let responseData;
 
-        const responseData = await requestQuickCache<any>({
-          url: `${tileApiEndpointToUse}/mosaic/register`,
-          payload,
-          controller
-        });
-
-        setMosaicUrl(responseData.links[1].href);
+        try {
+          responseData = await requestQuickCache<any>({
+            url: `${tileApiEndpointToUse}/searches/register`,
+            payload,
+            controller
+          });
+          const mosaicUrl = responseData.links[1].href;
+          setMosaicUrl(mosaicUrl.replace('/{tileMatrixSetId}', '/WebMercatorQuad'));
+        } catch (e) {
+          // @NOTE: conditional logic TO BE REMOVED once new BE endpoints have moved to prod... Fallback on old request url if new endpoints error with nonexistance... 
+          if (e.request) {
+            // The request was made but no response was received
+            responseData = await requestQuickCache<any>({
```

Comment:
Ah, I realized that we might need a nested try- because we cannot catch the error from this request. But then I realized that this try is already nested 😅(https://github.com/NASA-IMPACT/veda-ui/blob/80687e78127a3f57a998cd98646d4a6b591ad655/app/scripts/components/common/map/style-generators/raster-timeseries.tsx#L153-L210)  maybe we can move this to the first-level try-catch? https://github.com/NASA-IMPACT/veda-ui/blob/80687e78127a3f57a998cd98646d4a6b591ad655/app/scripts/components/common/map/style-generators/raster-timeseries.tsx#L195

Reply:
hah also just saw this.. i ended up throwing the error in nested try/catch so error can bubble up to the first try/catch.  I think this is fine, i'm wrapping the nested try catch directly around the request and catching and checking for a specific error right away => that errors either leads to old request or throws error back up. It will be easy to identify and clean up later too.

Reply:
oh yeah, I like this approach (https://github.com/NASA-IMPACT/veda-ui/pull/1075/files#diff-fc839f8286ca1783e679e0fcddd8a3da452570ebbe9f433e16fda7003b91687eR302) better.

As far as I can see, I don't think the error for this specific request will be caught because this request is already in 'catch' block unless we are wrapping it with try-catch again?

Reply:
Oh wow, I mistook how this would work; I guess the parental level of try-catch catches the error in the nested catch? 🤔 Thanks for giving it a try!